Description: Mark MemStick and MMC devices as such, so that they get automounted.
Origin: Backported from udisks2, http://cgit.freedesktop.org/udisks/commit/?id=a0df11b0b
Bug-Ubuntu: https://launchpad.net/bugs/942777

Index: udisks/data/80-udisks.rules
===================================================================
--- udisks.orig/data/80-udisks.rules	2011-11-11 16:44:35.952825890 +0100
+++ udisks/data/80-udisks.rules	2012-03-30 07:35:08.254419828 +0200
@@ -146,6 +146,11 @@
 SUBSYSTEMS=="usb", ENV{ID_MODEL}=="*SM_Reader*", ENV{ID_DRIVE_FLASH_SM}="1"
 SUBSYSTEMS=="usb", ENV{ID_MODEL}=="*MS_Reader*", ENV{ID_DRIVE_FLASH_MS}="1"
 
+# TODO: figure out if the drive supports SD and SDHC and what the current
+# kind of media is - right now we just assume SD
+KERNEL=="mmcblk[0-9]", SUBSYSTEMS=="mmc", ENV{DEVTYPE}=="disk", ENV{ID_DRIVE_FLASH_SD}="1", ENV{ID_DRIVE_MEDIA_FLASH_SD}="1"
+KERNEL=="mspblk[0-9]", SUBSYSTEMS=="memstick", ENV{DEVTYPE}=="disk", ENV{ID_DRIVE_FLASH_MS}="1", ENV{ID_DRIVE_MEDIA_FLASH_MS}="1"
+
 ##############################################################################################################
 
 # If previous rules did not set an icon, provide a default one for media players
